cloud-nio/cloud-nio-spi/src/main/scala/cloud/nio/spi/CloudNioRetry.scala

```diff
@@ -4,17 +4,18 @@ import com.typesafe.config.Config
 import net.ceedubs.ficus.Ficus._
 
 import scala.concurrent.duration._
+import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
 
 class CloudNioRetry(config: Config) {
 
-  def defaultMaxRetries: Int = config.getAs[Int]("default-max-retries").getOrElse(10)
+  def defaultMaxRetries: Int = config.getAs[Int]("default-max-retries").getOrElse(3)
 
   def isFatal(exception: Exception): Boolean = !isTransient(exception)
 
   def isTransient(exception: Exception): Boolean = false
 
-  def defaultBackOff: CloudNioBackoff = CloudNioSimpleExponentialBackoff(1.second, 60.seconds, 1.1)
+  def defaultBackOff: CloudNioBackoff = CloudNioSimpleExponentialBackoff(2 seconds, 8 seconds, 2)
```

Comment:
The total wait time is actually not too different between these two settings (~18 s vs ~14s), though the number of potential calls to Martha is much less.